doc/man3/SSL_extension_supported.pod

```diff
@@ -59,9 +97,11 @@ internally by OpenSSL and 0 otherwise.
 =head1 EXTENSION CALLBACKS
 
 The callback B<add_cb> is called to send custom extension data to be
-included in ClientHello for TLS clients or ServerHello for servers. The
-B<ext_type> parameter is set to the extension type which will be added and
-B<add_arg> to the value set when the extension handler was added.
+included in various TLS messages. The B<ext_type> parameter is set to the
```

Comment:
There was already some words in the NOTES section which said something along those lines. I moved it up into the main DESCRIPTION section so it is clearer.
@kaduk - it's a good question. I don't really know the history of this.